devel/mps_load_test.sh

```diff
@@ -215,11 +215,11 @@ make -j8 CONF=$compile_mode apache_trace_stress_test_server \
 
 # If a custom .so got specified, install it.
 if [[ -n "$custom_so" ]]; then
-  install -c $custom_so /usr/local/apache2/modules/mod_pagespeed.so
+  install -c $custom_so $HOME/apache2/modules/mod_pagespeed.so
```

Comment:
Well, the release process doc goes:

```
Please load-test using multiple Apache version & MPMs 
..... 
These can all be built using install/build_development_apache.sh, which builds a new ~/apache2
```

So it surprised me that `devel/mps_load_test.sh` pointed to a location that didn't exist on my machine. If we only load test in the release flow, this change works..